test/testHelpers.ts

```diff
@@ -0,0 +1,25 @@
+import { BATCH_TIME } from 'constants/'
+import BN from 'bn.js'
+
+export const DATE: Date = new Date('2017-04-23T08:30:15.500Z') // Epoch: 1492936215500 mills
+export const DATE_EPOCH = Math.floor(DATE.valueOf() / 1000) // 1492936215 sec
+export const BATCH_NUMBER = Math.floor(DATE_EPOCH / BATCH_TIME) // Math.floor(1492936215 / 300) = 4976454
+export const BATCH_SECOND = DATE_EPOCH % BATCH_TIME // 1492936215 % 300 = 15
+
+export const USER_1 = '0x0000000000000000000000000000000000000001'
+export const USER_2 = '0x0000000000000000000000000000000000000001'
+export const USER_3 = '0x0000000000000000000000000000000000000001'
+export const USER_4 = '0x0000000000000000000000000000000000000001'
+export const USER_5 = '0x0000000000000000000000000000000000000001'
```

Comment:
Note: all users have the same address

Reply:
:) thanks

It's not ready yet the PR, still a WIP, but didn't notice this yet!

Reply:
Sure, just letting you know :)

Reply:
Fixed: https://github.com/gnosis/dex-react/pull/31/commits/a77b8508140f4487ae29b92ef6f557afece16880